group_vars/cinder_api_container.yml

```diff
@@ -13,8 +13,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-container_skel:
-  repo_container:
-    properties:
-      service_name: repo
-      lxc_container_variant: "repo_server-{{ rpc_release }}"
+lxc_container_variant: "cinder-{{ rpc_release }}"
```

Comment:
This value was copied over from what was previously in the `rpcd/etc/openstack_deploy/env.d/cinder.yml` file. Should they be using the `*_all` groups instead?